phys/module_surface_driver.F

```diff
@@ -2966,7 +3001,7 @@ SUBROUTINE surface_driver(                                         &
      IF(SF_URBAN_PHYSICS.eq.1) THEN
        DO j=j_start(ij),j_end(ij)                             !urban
          DO i=i_start(ij),i_end(ij)                           !urban
-             IF( IVGTYP(I,J) == ISURBAN    .or. IVGTYP(I,J) == LCZ_1 .or. IVGTYP(I,J) == LCZ_2 .or. &
+           IF(IVGTYP(I,J) == ISURBAN   .or. IVGTYP(I,J) == LCZ_1 .or. IVGTYP(I,J) == LCZ_2 .or. &
```

Comment:
white space LCZ related only

Reply:
More efforts to clean up the bad indenting in the original code, in the areas that we specifically put in during the conflict resolution